sgkit/stats/aggregation.py

```diff
@@ -0,0 +1,55 @@
+import xarray as xr
+from xarray import DataArray, Dataset
+
+
+def allele_count(ds: Dataset) -> DataArray:
+    """Compute allele count from genotype calls
+
+    Parameters
+    ----------
+    ds : Dataset
+        Genotype call dataset such as from
+        `sgkit.create_genotype_call_dataset`
```

Comment:
```suggestion
        `sgkit.create_genotype_call_dataset`.
```


Nit, punctuation.

Reply:
Changed in https://github.com/pystatgen/sgkit/pull/36/commits/885a50d30f72d0c81ce56e01e3a13975fac9d02d

Reply:
Changed in https://github.com/pystatgen/sgkit/pull/36/commits/885a50d30f72d0c81ce56e01e3a13975fac9d02d